ansible/workers.yml

```diff
@@ -126,7 +126,10 @@
     package:
       name:
       - python2-jinja2  # needed for the new SSG yaml/jinja2 port
-      - python2-pytest  # needed for unit tests + coverage
+      - python2-pytest
+      - python3-pytest  # needed for unit tests + coverage
```

Comment:
something wonky happened here with the comments but I don't care enough to hold up this PR because of it